src/promptflow-devkit/promptflow/_sdk/_orchestrator/run_submitter.py

```diff
@@ -140,7 +140,7 @@ def _validate_inputs(cls, run: Run):
             raise UserErrorException(message=str(error), error=error)
 
     def _submit_bulk_run(
-        self, flow: Union[Flow, FlexFlow, Prompty], run: Run, local_storage: LocalStorageOperations
+        self, flow: Union[Flow, FlexFlow, Prompty], run: Run, local_storage: LocalStorageOperations, **kwargs
```

Comment:
why do we need to add this? what's the arg we are trying to pass here?

Reply:
It's for passing the "batch_use_async" argument from pf.run to the batch engine. 

This argument's value is determined by the callable type (async or sync) of each evaluator, please refer to Ln38 in `proxy_client.py`.